koin-projects/koin-core/src/main/kotlin/org/koin/ext/KoinPropertyExt.kt

```diff
@@ -0,0 +1,63 @@
+package org.koin.ext
```

Comment:
This file isn't complete enough.
On the previous version `getProperty` was accessible from KoinScope as well.

For example: 

```
single {
    ConnectionPool (
        maxConnections = getIntProperty("CONNECTION_POOL__MAX_CONNECTIONS")
    )
}
```

is not going to work, but should IMHO.